automated/linux/ltp-open-posix/ltp-open-posix.sh

```diff
@@ -103,4 +125,9 @@ else
     install_ltp_open_posix
 fi
 info_msg "Running run-ltp-open-posix"
-run_ltp_open_posix
+
+if [ "${GRP_TEST}" = "TMR" ]; then
```

Comment:
Why are the timers tests treated differently? We should be able to run any group from the open POSIX test suite.  This patch should be reworked to make it more general and not specific to the timers tests.